dbt/include/clickhouse/macros/utils/utils.sql

```diff
@@ -1,41 +1,51 @@
 {% macro clickhouse__get_test_sql(main_sql, fail_calc, warn_if, error_if, limit) -%}
-    {% set main_sql_formatted = clickhouse__place_limit(main_sql, limit) if limit !=None else main_sql%}
+    {% set main_sql_formatted = (
+        clickhouse__place_limit(main_sql, limit)
+        if limit != None
+        else main_sql
+    ) %}
     select
-      {{ fail_calc }} as failures,
-      {{ fail_calc }} {{ warn_if }} as should_warn,
-      {{ fail_calc }} {{ error_if }} as should_error
-    from (
-      {{ main_sql_formatted }}
-    ) dbt_internal_test
+        {{ fail_calc }} as failures,
+        {{ fail_calc }} {{ warn_if }} as should_warn,
+        {{ fail_calc }} {{ error_if }} as should_error
+    from ({{ main_sql_formatted }}) dbt_internal_test
 
 {%- endmacro %}
 
 
--- This macro is designed to add a LIMIT clause to a ClickHouse SQL query while preserving any ClickHouse settings specified in the query.
+-- This macro is designed to add a LIMIT clause to a ClickHouse SQL query while
```

Comment:
Hi @3fonov
Thank you for your contribution!

I see that you indented many lines. There is a lint process configured in a Makefile. Please return back the old format or alternatively run make lint in the project root dir to see what you need to adjust back.